app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -191,6 +199,63 @@ private fun RecentVisitItem(
     }
 }
 
+/**
+ * A recently visited history item.
+ *
+ * @param recentVisit The [RecentHistoryHighlight] to display.
+ * @param showDividerLine Whether to show a divider line at the bottom.
+ * @param onRecentVisitClick Invoked when the user clicks on a recent visit.
+ */
+@Composable
+private fun RecentlyVisitedHighlight(
```

Comment:
```suggestion
private fun RecentlyVisitedHistoryHighlight(
```
I don't know if this adds clarity, maybe?

Reply:
Inside the same same file I'd say the impact is small but this aligns the naming to the other `RecentlyVisitedHistoryGroup` composable. Thanks!